client/landing/gutenboarding/onboarding-block/create-site/style.scss

```diff
@@ -1,83 +1,54 @@
 @import 'assets/stylesheets/gutenberg-base-styles';
 @import '../../mixins';
+@import '../colors';
 
 $progress-step-title-height: 40px;
 $progress-step-number-step-height: 45px;
 $progress-duration: 800ms;
 $progress-widget-height: 300px;
 
-.create-site__background {
-	position: fixed;
-	top: 0;
-	left: 0;
-	right: 0;
-	bottom: 0;
-	z-index: 50;
-	background-color: $white;
+.create-site {
```

Comment:
Diff for this css is pretty hard to understand, basically I've flattened the html and shifted the nesting here up two? levels, very little css was changed.